src/locale/locales/de/messages.po

```diff
@@ -192,37 +187,33 @@ msgstr ""
 msgid "Appeal Content Warning"
 msgstr ""
 
-#: src/view/com/modals/AppealLabel.tsx:65
-#~ msgid "Appeal Decision"
-#~ msgstr "Berufungsentscheidung"
-
 #: src/view/com/util/moderation/LabelInfo.tsx:52
 msgid "Appeal this decision"
 msgstr "Einspruch gegen diese Entscheidung"
 
 #: src/view/com/util/moderation/LabelInfo.tsx:56
 msgid "Appeal this decision."
-msgstr "Einspruch gegen diese Entscheidung"
+msgstr "Einspruch gegen diese Entscheidung."
```

Comment:
(see above) Since there seems to be a detailed text here, it should be formulated accordingly "Einspruch gegen dieses Entscheidung einlegen."